assets/js/src/listing/notices.jsx

```diff
@@ -0,0 +1,111 @@
+import MailPoet from 'mailpoet';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import ReactStringReplace from 'react-string-replace';
+
+const resumeMailerSending = () => {
+  MailPoet.Ajax.post({
+    api_version: window.mailpoet_api_version,
+    endpoint: 'mailer',
+    action: 'resumeSending',
+  }).done(() => {
+    MailPoet.Notice.success(MailPoet.I18n.t('mailerSendingResumedNotice'));
+    window.mailpoet_listing.forceUpdate();
+  }).fail((response) => {
+    if (response.errors.length > 0) {
+      MailPoet.Notice.error(
+        response.errors.map(error => error.message),
+        { scroll: true }
+      );
+    }
+  });
+};
+
+function MailerError(props) {
+  if (props.mta_log.error && props.mta_log.status === 'paused' && props.mta_log.error.operation !== 'authorization') {
+    if (props.mta_log.error.operation === 'migration') {
+      return (
+        <div className="mailpoet_notice notice notice-warning">
```

Comment:
So here we can't use `MailPoet.Notice` so we have to imitate it? That's really not nice but if there's no other option at the moment, what can we do.

Reply:
We are not imitating `MailPoet.Notice` here, I kept the CSS classes just to keep the same display.